src/main/java/frc/robot/subsystems/elevator/ElevatorIOReal.java

```diff
@@ -1,15 +1,96 @@
 package frc.robot.subsystems.elevator;
 
+import static edu.wpi.first.units.Units.Degrees;
+import static edu.wpi.first.units.Units.Meters;
+
+import com.ctre.phoenix6.controls.DutyCycleOut;
+import com.ctre.phoenix6.controls.MotionMagicVoltage;
+import com.ctre.phoenix6.controls.StrictFollower;
+import com.ctre.phoenix6.hardware.TalonFX;
 import edu.wpi.first.units.Distance;
 import edu.wpi.first.units.MutableMeasure;
+import edu.wpi.first.units.Units;
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.Servo;
+import frc.robot.Ports;
 
 public class ElevatorIOReal implements ElevatorIO {
+    private final Servo servo;
+    private final TalonFX mainMotor;
+    private final TalonFX auxMotor;
+
+    private final DigitalInput bottomSensor;
+    private final DigitalInput topSensor;
```

Comment:
Again, should be wired directly to the motor controller.

Reply:
36b85ab1ea79db7c196561265572b3c460aa3bae